lobby/src/main/java/org/triplea/lobby/server/ModeratorController.java

```diff
@@ -350,10 +350,11 @@ public String getInformationOn(final INode node) {
     builder.append("\r\nHashed Mac: ");
     if (UNKNOWN_HASHED_MAC_ADDRESS.equals(mac)) {
       builder.append("(Unknown)");
-    } else if (MacFinder.isValidHashedMacAddress(mac)) {
-      builder.append(MacFinder.trimHashedMacAddressPrefix(mac));
```

Comment:
This is the functional change that no longer strips the hashed MAC address prefix when showing player information.